LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018, Richard Feldman
+Copyright (c) 2019, Richard Feldman
```

Comment:
Something I learned recently is that [it's best practice to leave the oldest applicable copyright date](https://stackoverflow.com/a/2391555); updating it actually weakens the copyright claim!

```suggestion
Copyright (c) 2018, Richard Feldman
```

Reply:
Interesting, makes sense though.